### DISCUSSION TITLE
Display related contents

### DISCUSSION THREAD

**Discussion by carolin-skiply** [directus/directus#13487] [Archive]
Hi there,

I am looking to display some related contents of one of my content collection in one single view.

Here is the context:
- I have several collections:
  - `places` = a location, a site
  - `clocks` = clock-in or out of my collaborators
  - `late_passage` = notification generated if my collaborators are late on site
- data for clock-in and clock-out are sent to my collection `clocks`. Each clock (in or out) is associated to a place
- late notifications (`late_passage`) are generated when a collaborator is late. Each notification is associated to a place as well.
I'd like to show these related clocks (in and out) and late notifications when I am consulting the detail view of a place. 
For instance, the detail view could layout as: My place Name then the list of clock-in and out for this place and the list of late notifications, maybe some indicators, and (not mandatory) a crayon button to modify the actual field associated to the place (address, phone, ...).

Is that possible with the detail view of my collection `places`? using interfaces? should I create a new module? can I do this another way?

Thanks for the help,
Caroline.

**Comment by rijkvanzanten** [comment]
You should be able to configure the fields on the `places` page to have the detail page render as you're describing 🙂 Sounds like you'd setup some one-to-many fields from `places` to `clocks` and `late_passage` respectively to list those events per place

**Reply by carolin-skiply** [reply]
I'll try that. Thks.
